integration/map.go

```diff
@@ -85,7 +86,7 @@ func RunMapIntegration(ctx context.Context, mapID int64, client trillian.Trillia
 
 	// Check values
 	// Mix up the ordering of requests
-	h := merkle.NewMapHasher(testonly.Hasher)
+	h := rfc6962.DefaultHasher
```

Comment:
The rfc6962 hasher is the default hasher with 0 dependencies.  It is used both for the log and the map at this point. 

I thought it would be more correct to have the test explicitly declare what hash function it wants rather than relying on an indirection through `testonly`.